libr/core/cmd_print.c

```diff
@@ -2853,11 +2881,12 @@ static int cmd_print_blocks(RCore *core, const char *input) {
 						   total.flags, total.functions, total.comments, total.symbols, total.strings);
 			r_cons_printf ("`-------------'----------------------------'\n");
 			break;
+		case 'e':
 		default:
 			if (use_color) {
 				r_cons_print (Color_RESET);
 			}
-			r_cons_printf ("] 0x%"PFMT64x "\n", to);
+			r_cons_printf ("] 0x%08"PFMT64x "\n", to);
```

Comment:
> It makes no sense to break anything. Its a new command

It broke because both p- and p-e share the code for printing offsets in lines 2766, 2889.